src/pages/registration/RegistrationForm.tsx

```diff
@@ -86,11 +83,10 @@ const RegistrationForm = ({ identifier, isNewRegistration }: RegistrationFormPro
     <StyledRegistration>
       <SkipLink href="#form">{t('common:skip_to_schema')}</SkipLink>
       <Formik
-        enableReinitialize
```

Comment:
Mener denne ble lagt inn for å støtte at oppdatert registrering ble lagt inn som initValues når man lagrer, men det trengs ikke lenger siden vi nå viser spinner mens det laster, slik at dette mountes på nytt etter lagring uansett. 95% sikker